src/Tribe/Views/V2/View.php

```diff
@@ -755,25 +755,37 @@ public function next_url( $canonical = false, array $passthru_vars = [] ) {
 
 		$url = $this->get_url();
 
+		$query_args = [];
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 441593dbe34e42e31b9d91393ccea923</sub>